Author: Hilko Bengen <bengen@debian.org>
Description: urlsnarf: use timestamps from pcap file if available. Closes: #573365
Index: dsniff-2.4b1+debian/urlsnarf.c
===================================================================
--- a/urlsnarf.c
+++ b/urlsnarf.c
@@ -57,7 +57,7 @@
 {
 	static char tstr[32], sign;
 	struct tm *t, gmt;
-	time_t tt = time(NULL);
+	time_t tt = nids_last_pcap_header->ts.tv_sec;
 	int days, hours, tz, len;
 	
 	gmt = *gmtime(&tt);
